javascript/javascript3/week3/homework-js3-week3/screenshotss.js

```diff
@@ -0,0 +1,141 @@
+const BASE_URL_SCREENSHOT_GENERATOR = 'https://website-screenshot6.p.rapidapi.com';
+const API_HOST_SCREENSHOT_GENERATOR = 'website-screenshot6.p.rapidapi.com';
+
+const BASE_URL_CRUD_API = `https://crudcrud.com/api/${API_KEY_CRUD_API}`;
+
+const input = document.getElementsByClassName('url-input')[0];
+const button = document.getElementsByClassName('get-screenshot-button')[0];
+const image = document.getElementsByClassName('screenshot')[0];
+const allScreenshotsContainer = document.getElementsByClassName('all-screenshots')[0];
+const showAllScreenshotsButton = document.getElementsByClassName('get-list')[0];
+const screenshotContainer = document.getElementsByClassName('screenshot-container')[0];
+const mainContainer = document.getElementsByClassName('main-container')[0];
+const preloader = document.getElementsByClassName('loading')[0];
+
+let screenshotsList = [];
+
+/**
```

Comment:
Great that you are considering documentation, it is really important for code readability. Have you tried chatGPT to get some inspiration on how to write documentation for your code?